app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -42,6 +43,8 @@
 
     private SwipeRefreshLayout swipeRefreshLayout;
 
+    private TextView noSponsors;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSponsors' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594560/issues/source?bid=3233850&fileBranchId=3233896#l46)